apps/challenges/models.py

```diff
@@ -228,6 +228,7 @@ class StarChallenge(TimeStampedModel):
     """
     user = models.ForeignKey(User)
     challenge = models.ForeignKey(Challenge)
+    is_starred = models.BooleanField(default=False)
```

Comment:
We have done this so that we can get information in future if we need some analytics for this. :)